db/migrate/20211211083901_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The migration is fine except for one thing. The user should not be directly referenced to the comment object instead with a different name (`author`) than the model name. To do so the migration should have something like this. 
```
`t.references :author, foreign_key: { to_table: 'users' }`
```
Kindly use the proper migration to remove the current migration and add another one. Do not directly edit migration files. :pray: 
Please carefully look at the ERD. 
https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png